zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -499,19 +499,19 @@ public InterpreterGroup createInterpreterGroup(String id, InterpreterOption opti
     AngularObjectRegistry angularObjectRegistry;
 
     InterpreterGroup interpreterGroup = new InterpreterGroup(id);
-    if (option.isRemote()) {
-      angularObjectRegistry = new RemoteAngularObjectRegistry(
-          id,
-          angularObjectRegistryListener,
-          interpreterGroup
-      );
-    } else {
-      angularObjectRegistry = new AngularObjectRegistry(
-          id,
-          angularObjectRegistryListener);
-
-      // TODO(moon) : create distributed resource pool for local interpreters and set
-    }
+//    if (option.isRemote()) {
+    angularObjectRegistry = new RemoteAngularObjectRegistry(
+        id,
+        angularObjectRegistryListener,
+        interpreterGroup
+    );
+//    } else {
+//      angularObjectRegistry = new AngularObjectRegistry(
+//          id,
+//          angularObjectRegistryListener);
+//
+//      // TODO(moon) : create distributed resource pool for local interpreters and set
+//    }
```

Comment:
@felixcheung Even though github shows the changes but I thought it's easy to find out what the changes are. I'll remove them after review.